catalyst/utils/typing.py

```diff
@@ -1,12 +1,23 @@
-from typing import Union  # isort:skip
+from typing import Union, Dict  # isort:skip
 
 import torch
 from torch import nn, optim
+from torch.utils import data
 
-Model = nn.Module
+Model = Union[nn.Module, Dict[str, nn.Module]]
```

Comment:
i'm not sure about this. This line makes the `Model` unclear. I would prefer to use `Union[nn.Module, Dict[str, nn.Module]]`  where it's needed explicitly

Reply:
+1
Now I understand why you write `model: Model` everywhere, but same as Roman I do not think that redefine `Model` in such a way is a good idea

Reply:
Reverted it